CHANGELOG.rst

```diff
@@ -8,12 +8,12 @@ the version control of each dependency.
 28.0.0 (unreleased)
 ===================
 
-**Changed**
+**Breaking Changes**
```

Comment:
Is this actually a breaking change? While we include the source code to `kinto` in a different manner (in `site-packages` instead of the `PYTHONPATH`), it's the same Remote Settings plugin, no?